build.gradle

```diff
@@ -91,6 +91,70 @@ repositories {
+    implementation "com.google.inject:guice:5.1.0"
+    implementation "org.opensearch.client:opensearch-java:${opensearch_version}"
+    implementation "org.opensearch.client:opensearch-rest-client:${opensearch_version}"
+    implementation "org.opensearch.client:opensearch-rest-high-level-client:${opensearch_version}"
+    implementation group: 'com.google.guava', name: 'guava', version:'31.0.1-jre'
+    implementation group: 'com.google.guava', name: 'failureaccess', version:'1.0.1'
+    implementation group: 'org.javassist', name: 'javassist', version:'3.28.0-GA'
+    implementation group: 'org.apache.commons', name: 'commons-math3', version: '3.6.1'
+    implementation group: 'com.google.code.gson', name: 'gson', version: '2.8.9'
+    implementation group: 'com.yahoo.datasketches', name: 'sketches-core', version: '0.13.4'
+    implementation group: 'com.yahoo.datasketches', name: 'memory', version: '0.12.2'
+    implementation group: 'commons-lang', name: 'commons-lang', version: '2.6'
+    implementation group: 'org.apache.commons', name: 'commons-pool2', version: '2.10.0'
+    implementation 'software.amazon.randomcutforest:randomcutforest-serialization:3.0-rc3'
+    implementation 'software.amazon.randomcutforest:randomcutforest-parkservices:3.0-rc3'
+    implementation 'software.amazon.randomcutforest:randomcutforest-core:3.0-rc3'
+
+    // force Jackson version to avoid version conflict issue
+//    implementation "com.fasterxml.jackson.core:jackson-core:${versions.jackson}"
+//    implementation "com.fasterxml.jackson.core:jackson-databind:${versions.jackson}"
+//    implementation "com.fasterxml.jackson.core:jackson-annotations:${versions.jackson}"
+
+    // used for serializing/deserializing rcf models.
+    implementation group: 'io.protostuff', name: 'protostuff-core', version: '1.8.0'
+    implementation group: 'io.protostuff', name: 'protostuff-runtime', version: '1.8.0'
+    implementation group: 'io.protostuff', name: 'protostuff-api', version: '1.8.0'
+    implementation group: 'io.protostuff', name: 'protostuff-collectionschema', version: '1.8.0'
+    implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.12.0'
+
+    implementation "org.jacoco:org.jacoco.agent:0.8.5"
+    implementation ("org.jacoco:org.jacoco.ant:0.8.5") {
+        exclude group: 'org.ow2.asm', module: 'asm-commons'
+        exclude group: 'org.ow2.asm', module: 'asm'
+        exclude group: 'org.ow2.asm', module: 'asm-tree'
+    }
+
+    testImplementation group: 'org.opensearch.test', name: 'framework', version:"${opensearch_version}"
+    testImplementation group: 'pl.pragmatists', name: 'JUnitParams', version: '1.1.1'
+    testImplementation group: 'org.mockito', name: 'mockito-core', version: '5.3.1'
+    testImplementation group: 'org.powermock', name: 'powermock-api-mockito2', version: '2.0.9'
+    testImplementation group: 'org.powermock', name: 'powermock-module-junit4', version: '2.0.9'
```

Comment:
I see that AD main is using powermock dependencies v2.0.2 instead of v2.0.9 [here](https://github.com/opensearch-project/anomaly-detection/blob/0a48da9e1a0cd70c64cc2da228674baaf377c89e/build.gradle#L144). What's the justification for this upgrade?

Reply:
No justification. Just a dependency upgrade :slightly_smiling_face: